src/bundle/Controller/RecommendationController.php

```diff
@@ -82,11 +68,6 @@ public function showRecommendationsAction(Request $request): Response
         );
     }
 
-    public function sendDeliveryFeedback(bool $value): void
```

Comment:
Have you checked if there are any external calls to this method?

Reply:
Of course, this is internal Controller and there are not any external calls.